l1macros/performances_nano.py

```diff
@@ -67,6 +78,20 @@ def main():
             config_file = '../config_cards/full_ZToMuMu.yaml'
         elif args.channel == 'ZToEE':
             config_file = '../config_cards/full_ZToEE.yaml'
+        elif args.channel == 'ZToTauTau':
+            config_file = '../config_cards/full_ZToTauTau.yaml'
```

Comment:
As mentioned above that file seems to be missing in fact.

Reply:
This config file needed for the offline DQM Offline selection is present (config_cards/full_ZToTauTau_DQMOff.yaml)